crucible-llvm-cli/test-data/isnan.cbl

```diff
@@ -0,0 +1,30 @@
+; Floats produced with Python:
+;
+;     from math import nan
+;     import struct
+;     def float_to_bin(num):
+;         bits, = struct.unpack('!I', struct.pack('!f', num))
+;         return hex(int("{:032b}".format(bits), base=2))
+;     print(float_to_bin(42.0))
+;     print(float_to_bin(nan))
+
+(defun @main () Unit
+  (start start:
+    (let g (resolve-global "isnan"))
+    (let h (load-handle (Ptr 32) ((FP Double)) g))
+
+    (let i42 (bv 64 0x42280000))
```

Comment:
I don't think this is right. `0x42280000` is the bit-level representation of `42.0` when represented as a single-precision float (i.e., 32 bits), not a double-precision float (i.e., 64 bits). You can see this in GHCi:

```
λ> :m + GHC.Float Numeric
λ> showString "0x" $ showHex (castFloatToWord32 42.0) ""
"0x42280000"
λ> showString "0x" $ showHex (castDoubleToWord64 42.0) ""
"0x4045000000000000"
```

Similar comments apply to the NaN values below.